doc/content/devices/adding-devices/_index.md

```diff
@@ -14,41 +14,99 @@ This section contains instructions for adding devices in {{% tts %}}.
 
 ## Adding Devices using the Console
 
-Go to **Devices** in the left menu and click on **+ Add Device** to reach the end device registration page. Fill the device ID, the LoRaWAN MAC and PHY versions and the frequency plan used by the device.
+Devices are managed under applications. An application can contain an unlimited number of devices, but it can be helpful to sort devices in to applications by function or geographical area, to make the integrations and live data views more useful.
```

Comment:
This should be moved up (above this title) because it refers to devices and applications in general